packages/schematics/schematics/schematic/files/src/my-full-schematic/index.ts

```diff
@@ -0,0 +1,21 @@
+import {
+    Rule,
+    SchematicContext,
+    Tree,
+    chain,
+    schematic,
+} from '@angular-devkit/schematics';
+
+export default function (options: any): Rule {
+    // The chain rule allows us to chain multiple rules and apply them one after the other.
+    return chain([
+        (_tree: Tree, context: SchematicContext) => {
+            // Show the options for this Schematics.
+            context.logger.info('My Full Schematic: ' + JSON.stringify(options));
+        },
+        // The schematic Rule calls the schematic from the same collection, with the options
+        // passed in. Please note that if the schematic has a schema, the options will be
+        // validated and could throw, e.g. if a required option is missing.
+        schematic('my-other-schematics', { option: true }),
```

Comment:
`my-other-schematics` should be `my-other-schematic` to match the name in `collection.json`

Reply:
Done.